samples/03.live-inking/src/main-view.ts

```diff
@@ -0,0 +1,334 @@
+/**
```

Comment:
Would you be open to splitting this file into 2 or more? File is quite long, over 300 lines already.